core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1226,6 +1234,18 @@ protected function panel()
 			return '';
 		}
 
+		// Compile limit menu if placeholder is present
+		foreach ($arrPanels as $key => $strPanel)
+		{
+			if (strpos('###limit_menu###', $strPanel) === false)
+			{
+				continue;
+			}
+
+			$limitPanel = $this->limitMenu();
+			$arrPanels[$key] = str_replace('###limit_menu###', $limitPanel, $strPanel);
```

Comment:
```suggestion
			$arrPanels[$key] = str_replace('###limit_menu###', $this->limitMenu(), $strPanel);
```

No need for a variable here, is there?